tap.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for tap 14.4.2
```

Comment:
Not a great idea to hard-code the version in there.  It's already out of date ;)

Reply:
Yeah, I guess it doesn't make sense to specify version if it ships with the package, especially with the tests they are supposed to be up-to-date ;) It is just a normal practice to hard-code version when typings are external so it was there from the template.